sleep_for_days/activities.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+
+from temporalio import activity
+
+
+@dataclass
+class SendEmailInput:
+    email_msg: str
+
+
+@activity.defn()
+async def send_email(input: SendEmailInput) -> str:
+    """
+    A stub Activity for sending an email.
+    """
+    result = f"Email message: {input.email_msg}, sent"
+    print(result)
```

Comment:
Arguably since this output is coming from the worker process, it should use logging. (I think the print statement is fine in the client/starter process.)